server/middleware/products.coffee

```diff
@@ -16,6 +16,7 @@ get = wrap (req, res) ->
     )
     return res.send(products)
 
+  # Remove old unsupported subscription products
   products = _.filter products, (p) -> p.name not in ['year_subscription', 'lifetime_subscription2']
```

Comment:
Doesn't need changing, but do you think this is better as a blacklist or whitelist going forward?